pyproject.toml

```diff
@@ -53,16 +54,17 @@ doc = [
     "jupyter_sphinx==0.4.0",
     "jupyterlab==4.0.5",
     "numpydoc==1.5.0",
-    "pandas==2.1.0",
```

Comment:
You dont depend on pandas anymore??  But yes on scipy? just double checking.

Reply:
`scipy` is used in the examples.
`pandas` in not used in `ansys-math-core`.